INTEGRATE.md

```diff
@@ -1,31 +1,35 @@
 # Local integration between Porta/System with Keycloak through Zync
 Install these 3 components to have a complete integration of managed SSO applications in Porta.
 
-## Install Porta
+1. Install Porta
 Follow [the installation guide](https://github.com/3scale/porta/blob/master/INSTALL.md).
 
-## Install Zync
+2. Install Zync
 Follow [the installation guide](https://github.com/3scale/zync/blob/master/INSTALL.md).
 
-## Install Keycloak
-To have a keycloak running locally, do `docker run --name keycloak -d -p 8080:8080 -e KEYCLOAK_USER=admin -e KEYCLOAK_PASSWORD=p jboss/keycloak`
```

Comment:
Instruction in one line.
`commands` in the next line(s).

Reply:
outdated, old version 😛